recipe/meta.yaml

```
@@ -23,22 +23,23 @@ requirements:
     - sphinx 5.0.2
   run:
     - python
-    - sphinx >=5,<8
+    - sphinx >=5,<9
```

```suggestion
    - sphinx >=6,<9
```
https://github.com/executablebooks/sphinx-design/blob/v0.6.1/pyproject.toml#L31